src/runescape/bestiary.ts

```diff
@@ -0,0 +1,238 @@
+import got from "got"
+import { bestiary } from "../configs/runescape"
+import {
+  Area,
+  Beast,
+  BeastSearchResult,
+  SlayerCategory,
+  Weakness,
+} from "../lib/RuneScape"
+import { Jagex, Letter } from "../types"
+
+export const getAreas = async () => {
+  try {
+    const areas = await got(bestiary.endpoints.areas).json<
+      Jagex.Bestiary.Areas
+    >()
```

Comment:
Why have generics on a new line?

Reply:
Prettier does the formatting on this one. Didn't give it any thought.